wasmcloud-chat/wasmcloudchat/config/config.exs

```diff
@@ -0,0 +1,31 @@
+# This file is responsible for configuring your application
+# and its dependencies with the aid of the Mix.Config module.
+#
+# This configuration file is loaded before any dependency and
+# is restricted to this project.
+
+# General application configuration
+use Mix.Config
+
+config :wasmcloudchat,
+  ecto_repos: [Wasmcloudchat.Repo]
+
+# Configures the endpoint
+config :wasmcloudchat, WasmcloudchatWeb.Endpoint,
+  url: [host: "localhost"],
+  secret_key_base: "Zt1aw+xy6lhkkPNaEI2DyYviTpj0wkoNwTcYdyr5yuV/AjUKNbmEeDXeF8Ri7NN3",
+  render_errors: [view: WasmcloudchatWeb.ErrorView, accepts: ~w(html json), layout: false],
+  pubsub_server: Wasmcloudchat.PubSub,
+  live_view: [signing_salt: "V/6VN3DC"]
```

Comment:
I don't know what this is exactly for but I saw `secret_key` and wasn't sure if it should be committed, is this all fine?

Reply:
this is the session encryption key for the web app, and it's fine to be committed since the production version is overwritten with an environment variable.